logic/broadcaster.js

```diff
@@ -103,27 +103,16 @@ class Broadcaster {
 	 */
 	getPeers(params, cb) {
 		params.limit = params.limit || this.config.peerLimit;
-		params.broadhash = params.broadhash || null;
-		params.normalized = false;
 
-		const originalLimit = params.limit;
-		modules.peers.list(params, (err, peers) => {
-			if (err) {
-				return setImmediate(cb, err);
-			}
+		const peers = library.logic.peers.listRandomConnected(params);
 
-			if (originalLimit === constants.maxPeers) {
-				library.logger.info(
-					[
-						'Broadhash consensus now',
-						modules.peers.getLastConsensus(),
-						'%',
-					].join(' ')
-				);
-			}
+		library.logger.info(
```

Comment:
Removing `if (originalLimit === constants.maxPeers) {` condition will not cause spam with that log?